src/Tools/dotnet-monitor/CollectionRules/Options/Triggers/EventCounterOptions.Validate.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options.Triggers
+{
+    partial class EventCounterOptions : IValidatableObject
+    {
+        IEnumerable<ValidationResult> IValidatableObject.Validate(ValidationContext validationContext)
+        {
+            List<ValidationResult> results = new();
+
+            if (!GreaterThan.HasValue && !LessThan.HasValue)
```

Comment:
It looks like GreaterThan and LessThan are both just doubles; should we be enforcing non-negative?

Reply:
I don't think we have a need to enforce that. While I haven't seen any of the counters report negative numbers, I don't think there is any reason why they couldn't.